docs/content/theme/color-palette.md

```diff
@@ -37,15 +42,16 @@ A user interface uses color to convey:
  - **Information**: Charts, graphs, and wayfinding elements
  - **Hierarchy**: Showing structured order through color and typography
 
-We've exposed 3 color variants for this design system to represent your brand:
+We've exposed 2 color variants for this design system to represent your brand:
```

Comment:
Should we note that this is specific to UI since complement still exists for surfaces?

Reply:
updated, thanks